website/docs/r/template.html.markdown

```diff
@@ -106,10 +106,12 @@ The following arguments are supported:
 
 `disk` supports the following arguments
 
-* `image_id` - (Required) ID of the image to attach to the virtual machine.
+* `image_id` - (Optional) ID of the image to attach to the virtual machine. Conflicts with `type` and `format`.
 * `size` - (Optional) Size (in MB) of the image attached to the virtual machine. Not possible to change a cloned image size.
 * `target` - (Optional) Target name device on the virtual machine. Depends of the image `dev_prefix`.
 * `driver` - (Optional) OpenNebula image driver.
+* `volatile_type` - (Optional) Type of the disk: `swap` or `fs`. Type `swap` is not supported in vcenter. Conflicts with `image_id`.
+* `format` - (Optional) Format of the Image: `raw` or `qcow2`. Conflicts with `image_id`.
```

Comment:
it is the format for the volatile disk, not the image.

"Format of the volatile disk".

I sugges renaming `format` by `volatile_disk_format`

Reply:
I changed for volatile_format, is it good ?